recipe/meta.yaml

```diff
@@ -30,51 +31,49 @@ requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    - patch   # [not win]
+    - m2-patch    # [win]
   host:
+    - pip
+    - python
     - cython
+    - docutils
```

Comment:
Is `docutils` still required? See https://github.com/biolab/orange3/commit/78f7b30a89a7b78ef6442f0b828f41a6f5c27951